resources/sts/4.11/sts_installer_core_permission_policy.json

```diff
@@ -0,0 +1,173 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+		    "autoscaling:DescribeAutoScalingGroups",
+		    "ec2:AllocateAddress",
+		    "ec2:AssociateAddress",
+		    "ec2:AttachNetworkInterface",
+		    "ec2:AuthorizeSecurityGroupEgress",
+		    "ec2:AuthorizeSecurityGroupIngress",
+		    "ec2:CopyImage",
+		    "ec2:CreateNetworkInterface",
+		    "ec2:CreateSecurityGroup",
+		    "ec2:CreateTags",
+		    "ec2:CreateVolume",
+		    "ec2:DeleteNetworkInterface",
+		    "ec2:DeleteSecurityGroup",
+		    "ec2:DeleteSnapshot",
+		    "ec2:DeleteTags",
+		    "ec2:DeleteVolume",
+		    "ec2:DeregisterImage",
+		    "ec2:DescribeAccountAttributes",
+		    "ec2:DescribeAddresses",
+		    "ec2:DescribeAvailabilityZones",
+		    "ec2:DescribeDhcpOptions",
+		    "ec2:DescribeImages",
+		    "ec2:DescribeInstanceAttribute",
+		    "ec2:DescribeInstanceCreditSpecifications",
+		    "ec2:DescribeInstances",
+		    "ec2:DescribeInstanceStatus",
+		    "ec2:DescribeInstanceTypeOfferings",
+		    "ec2:DescribeInstanceTypes",
+		    "ec2:DescribeInternetGateways",
+		    "ec2:DescribeKeyPairs",
+		    "ec2:DescribeNatGateways",
+		    "ec2:DescribeNetworkAcls",
+		    "ec2:DescribeNetworkInterfaces",
+		    "ec2:DescribePrefixLists",
+		    "ec2:DescribeRegions",
+		    "ec2:DescribeReservedInstancesOfferings",
+		    "ec2:DescribeRouteTables",
+		    "ec2:DescribeSecurityGroups",
+		    "ec2:DescribeSubnets",
+		    "ec2:DescribeTags",
+		    "ec2:DescribeVolumes",
+		    "ec2:DescribeVpcAttribute",
+		    "ec2:DescribeVpcClassicLink",
+		    "ec2:DescribeVpcClassicLinkDnsSupport",
+		    "ec2:DescribeVpcEndpoints",
+		    "ec2:DescribeVpcs",
+		    "ec2:DisassociateRouteTable",
```

Comment:
We can remove this, it should only be required for VPCs

Reply:
Removed.